build.gradle

```diff
@@ -99,39 +116,39 @@ clean {
   doLast {
     exec {
       environment makeEnvironment
-      commandLine bash
-      args '-c', 'source ./set_env.sh && rm -rf 3rd_party/eigen cmake_build'
+      commandLine shell
+      args shell_flag, "${set_env} && set &&  + ${clean_cmd}"
```

Comment:
The `set` command in here should be removed before merging.  Also, is the `+` a typo?

Reply:
Yep, the `+` was a hangnail from the previous construction of the commandline